### PR TITLE
Enhanced FAQ

### PR DESCRIPTION
**Pull request by Soitora**
Fixed capitalization for "Getting started", added "Translation" and "Source problems", fixed stuff in "Reading local manga".

This branch has a link which is dependant on #21.

### CODE REVIEW

File: src/.vuepress/config.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ module.exports = {
 			'/',
 			{
 				title: 'Guides',
-				children: ['/help/guides/getting-started', '/help/guides/reading-local-manga']
+				children: ['/help/guides/getting-started', '/help/guides/reading-local-manga', '/help/guides/source-problems', '/help/guides/translation']
```
Comment (arkon) [inline src/.vuepress/config.js]: This is getting pretty long. Put each of them on a new line.

File: src/help/faq/extensions/README.md [hunk 2]
```diff
@@ -231,6 +189,50 @@ out which extension to install for the specific source that you want.
    - NANI? Scans
 :::
 
+## Source problems
```
Comment (arkon) [inline src/help/faq/extensions/README.md]: Part of this duplicates the `Source problems` page, right?

File: src/help/guides/translation.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+title: Translation
+lang: en-US
+---
+
+# Translation
```
Comment (arkon) [inline src/help/guides/translation.md]: This is mentioned elsewhere on the site too. Make sure to remove that, or direct that here.

File: src/.vuepress/config.js (hunk 1 above)
Comment (Soitora) [reply src/.vuepress/config.js]: Will push a commit fixing this in a sec.

File: src/help/faq/extensions/README.md (hunk 2 above)
Comment (Soitora) [reply src/help/faq/extensions/README.md]: This is hard to resolve, but I did my best in a commit I'm pushing soon. It's written by different people and it's vastly different in content but they kind of aim towards the same goal.

File: src/help/guides/translation.md (hunk 3 above)
Comment (Soitora) [reply src/help/guides/translation.md]: It was a one-liner, I changed the direct link to the guide page.

**Pull request merged**